### PR TITLE
April 4 executive copy

### PR DESCRIPTION
**Pull request by MakerJanSky**
One last confirmation to chase re: Ethena, DO NOT MERGE UNTIL $ IS ADDRESSED

### CODE REVIEW

File: governance/votes/Executive Vote - April 4, 2024.md [hunk 1]
```diff
@@ -0,0 +1,92 @@
+---
+title: Template - [Executive Vote] Increase GSM Pause, Increase Spark MetaMorpho Vault Debt Ceiling, Add Standard ilks to LINE_MOM, SparkLend Proxy Spell , and TACO Resolutions - April 4, 2024
```
Comment (0xldr) [inline governance/votes/Executive Vote - April 4, 2024.md:2]: :yellow_circle: **medium**

Improve accuracy and use human readable names

```suggestion
title: Template - [Executive Vote] Increase GSM Pause Delay, Increase Spark MetaMorpho Vault Maximum Debt Ceiling, Add Native Vaults to Debt Ceiling Breaker, SparkLend Proxy Spell, and TACO Resolutions - April 4, 2024
```

File: governance/votes/Executive Vote - April 4, 2024.md [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+summary: Increase the Governance Pause Delay, Increase Spark MetaMorph Maximum Debt Ceiling, Add non-PSM vaults to Debt Ceiling Breaker, Trigger SparkLendProxy Spell, and Approve BT Project Ethena Resolutions.
```
Comment (0xldr) [inline governance/votes/Executive Vote - April 4, 2024.md:3]: :yellow_circle: **medium**

1. fix typos
2. remove capitalization
3. improve clarity

```suggestion
summary: Increase the Governance Security Module Pause Delay, increase Spark MetaMorpho Maximum Debt Ceiling, add native vaults to Debt Ceiling Breaker, Trigger SparkLend Proxy Spell, and approve BT Project Ethena Resolutions. 
```

File: governance/votes/Executive Vote - April 4, 2024.md [hunk 3]
```diff
@@ -0,0 +1,92 @@
+---
+title: Template - [Executive Vote] Increase GSM Pause, Increase Spark MetaMorpho Vault Debt Ceiling, Add Standard ilks to LINE_MOM, SparkLend Proxy Spell , and TACO Resolutions - April 4, 2024
+summary: Increase the Governance Pause Delay, Increase Spark MetaMorph Maximum Debt Ceiling, Add non-PSM vaults to Debt Ceiling Breaker, Trigger SparkLendProxy Spell, and Approve BT Project Ethena Resolutions. 
+date: 2024-04-04T00:00:00.000Z
+address: "$spell_address"
+
+---
+# [Executive Proposal] Increase GSM Pause, Increase Spark MetaMorpho Vault Debt Ceiling, Add Standard ilks to LINE_MOM, TACO Resolutions, and SparkLend Proxy Spell - April 4, 2024
```
Comment (0xldr) [inline governance/votes/Executive Vote - April 4, 2024.md:8]: :yellow_circle: **medium**

see above

```suggestion
# [Executive Proposal] Increase GSM Pause Delay, Increase Spark MetaMorpho Vault Maximum Debt Ceiling, Add Native Vaults to Debt Ceiling Breaker, SparkLend Proxy Spell, and TACO Resolutions
```

File: governance/votes/Executive Vote - April 4, 2024.md [hunk 4]
```diff
@@ -0,0 +1,92 @@
...
+
+---
+
+## Executive Summary
+
+If this executive proposal passes, the following actions will occur within the Maker Protocol:
+
+- The [Governance Pause Delay (GSM)](https://manual.makerdao.com/parameter-index/core/param-gsm-pause-delay) will be increased to **16 hours**.
+- Spark MetaMorpho Vault DDM [Maximum Debt Ceiling (line)](https://manual.makerdao.com/module-index/module-dciam#maximum-debt-ceiling-line) will be increased to **1 billion DAI**.
+- Standard ilks will be added to the [Debt Ceiling Breaker (LINE_MOM)](https://manual.makerdao.com/governance-processes/executive-vote-considerations/gsm-exceptions#debt-ceiling-breaker) as detailed below.
```
Comment (0xldr) [inline governance/votes/Executive Vote - April 4, 2024.md:22]: :green_circle: **minor**

```suggestion
- Native Vault Engine ilks will be added to the [Debt Ceiling Breaker (LINE_MOM)](https://manual.makerdao.com/governance-processes/executive-vote-considerations/gsm-exceptions#debt-ceiling-breaker) as detailed below.
```

Comment (0xldr) [inline governance/votes/Executive Vote - April 4, 2024.md:22]: :green_circle: **minor**

Link to Scope definitions where possible

```suggestion
- The [Governance Pause Delay (GSM)](https://mips.makerdao.com/mips/details/MIP113#10-1-gsm-governance-security-module-pause-delay) will be increased to **16 hours**.
- Spark MetaMorpho Vault DDM [Maximum Debt Ceiling (line)](https://mips.makerdao.com/mips/details/MIP104#14-3-1-4-1-maximum-debt-ceiling-line-) will be increased to **1 billion DAI**.
- Standard ilks will be added to the [Debt Ceiling Breaker (LINE_MOM)](https://mips.makerdao.com/mips/details/MIP113#10-2-3-debt-ceiling-breaker) as detailed below.
```

File: governance/votes/Executive Vote - April 4, 2024.md [hunk 5]
```diff
@@ -0,0 +1,92 @@
...
+
+If you are new to voting in the Maker Protocol, please see the [voting guide](https://manual.makerdao.com/governance/voting-in-makerdao/on-chain-governance) to learn how voting works.
+
+---
+
+## Executive Summary
+
+If this executive proposal passes, the following actions will occur within the Maker Protocol:
+
+- The [Governance Pause Delay (GSM)](https://manual.makerdao.com/parameter-index/core/param-gsm-pause-delay) will be increased to **16 hours**.
```
Comment (0xldr) [inline governance/votes/Executive Vote - April 4, 2024.md:20]: :red_circle: **Blocking**

Incorrect value

```suggestion
- The [Governance Pause Delay (GSM)](https://manual.makerdao.com/parameter-index/core/param-gsm-pause-delay) will be increased to **30 hours**.
```

File: governance/votes/Executive Vote - April 4, 2024.md [hunk 6]
```diff
@@ -0,0 +1,92 @@
...
+If this executive proposal passes, The Governance Pause Delay (GSM) will be increased by 14 hours, from 16 to **30 hours**.
+
+### Spark MetaMorpho Vault DDM [Maximum Debt Ceiling (line)](https://manual.makerdao.com/module-index/module-dciam#maximum-debt-ceiling-line) Increase
+
+- **Authorization:** [Forum Post](https://forum.makerdao.com/t/morpho-spark-dai-vault-update-1-april-2024/24006/9)  
+- **Proposal:** [Forum post](https://forum.makerdao.com/t/morpho-spark-dai-vault-update-1-april-2024/24006)
+
+If this executive proposal passes, the Spark MetaMorpho Vault DDM Maximum Debt Ceiling (line) will increase by 900 million DAI, from 100 million to **1 billion DAI**.
+
+### Add ilks to [Debt Ceiling Breaker (LINE_MOM)](https://manual.makerdao.com/governance-processes/executive-vote-considerations/gsm-exceptions#debt-ceiling-breaker)
```
Comment (0xldr) [inline governance/votes/Executive Vote - April 4, 2024.md:52]: :green_circle: **minor**

See above

```suggestion
### Add ilks to [Debt Ceiling Breaker (LINE_MOM)](https://mips.makerdao.com/mips/details/MIP113#10-2-3-debt-ceiling-breaker)
```

File: governance/votes/Executive Vote - April 4, 2024.md [hunk 7]
```diff
@@ -0,0 +1,92 @@
...
+
+This executive proposal includes an office-hours modifier that means that it **can only be executed between 14:00 and 21:00 UTC, Monday - Friday**. 
+
+If this executive proposal does not pass within 30 days, then it will expire and can no longer have any effect on the Maker Protocol.
+
+---
+
+## Proposal Details
+
+### [Governance Pause Delay (GSM)](https://manual.makerdao.com/parameter-index/core/param-gsm-pause-delay) Increase
```
Comment (0xldr) [inline governance/votes/Executive Vote - April 4, 2024.md:38]: :green_circle: **minor**

See above

```suggestion
### [Governance Pause Delay (GSM)](https://mips.makerdao.com/mips/details/MIP113#10-1-gsm-governance-security-module-pause-delay) Increase
```

File: governance/votes/Executive Vote - April 4, 2024.md [hunk 8]
```diff
@@ -0,0 +1,92 @@
...
+
+- **Authorization:** [Poll](https://vote.makerdao.com/polling/QmcLsYwj)  
+- **Proposal:** [Forum post](https://forum.makerdao.com/t/gsm-pause-delay-increase-proposal/23929)
+
+If this executive proposal passes, The Governance Pause Delay (GSM) will be increased by 14 hours, from 16 to **30 hours**.
+
+### Spark MetaMorpho Vault DDM [Maximum Debt Ceiling (line)](https://manual.makerdao.com/module-index/module-dciam#maximum-debt-ceiling-line) Increase
```
Comment (0xldr) [inline governance/votes/Executive Vote - April 4, 2024.md:45]: :green_circle: **minor**

See above

```suggestion
### Spark MetaMorpho Vault DDM [Maximum Debt Ceiling (line)](https://mips.makerdao.com/mips/details/MIP104#14-3-1-4-1-maximum-debt-ceiling-line-) Increase
```

File: governance/votes/Executive Vote - April 4, 2024.md [hunk 9]
```diff
@@ -0,0 +1,92 @@
...
+
+- **Authorization:** [Forum Post](https://forum.makerdao.com/t/morpho-spark-dai-vault-update-1-april-2024/24006/9)  
+- **Proposal:** [Forum post](https://forum.makerdao.com/t/morpho-spark-dai-vault-update-1-april-2024/24006)
+
+If this executive proposal passes, the Spark MetaMorpho Vault DDM Maximum Debt Ceiling (line) will increase by 900 million DAI, from 100 million to **1 billion DAI**.
```
Comment (0xldr) [inline governance/votes/Executive Vote - April 4, 2024.md:50]: :green_circle: **minor**

Add missing unit

```suggestion
If this executive proposal passes, the Spark MetaMorpho Vault DDM Maximum Debt Ceiling (line) will increase by 900 million DAI, from 100 million DAI to **1 billion DAI**.
```

File: governance/votes/Executive Vote - April 4, 2024.md [hunk 10]
```diff
@@ -0,0 +1,92 @@
...
+* WBTC-A
+* WBTC-B
+* WBTC-C
+
+### Trigger SparkLend Proxy Spell
+
+- **Authorization:** Polls: [1](https://vote.makerdao.com/polling/QmdjqTvL), [2](https://vote.makerdao.com/polling/QmaEqEav), [3](https://vote.makerdao.com/polling/QmbCWUAP). [Forum Post](https://forum.makerdao.com/t/morpho-spark-dai-vault-update-1-april-2024/24006)  
+- **Proposal:** Forum posts: [1](https://forum.makerdao.com/t/mar-21-2024-proposed-changes-to-sparklend-for-upcoming-spell/23918), [2](https://forum.makerdao.com/t/morpho-spark-dai-vault-update-1-april-2024/24006)
+
+If this executive proposal passes, the SparkLend Proxy spell [0x7748C5E6EEda836247F2AfCd5a7c0dA3c5de9Da2](https://etherscan.io/address/0x7748c5e6eeda836247f2afcd5a7c0da3c5de9da2) will be triggered. In addition to polled changes on Activating the Kill Switch Security Module and increasing WBTC and ETH lending caps, this proxy spell contains changes suggested and approved in this [forum post](https://forum.makerdao.com/t/morpho-spark-dai-vault-update-1-april-2024/24006).
```
Comment (0xldr) [inline governance/votes/Executive Vote - April 4, 2024.md:73]: :yellow_circle: **Medium**

We strongly advise that Spark spell items are itemized for the benefit of voters like in previous docs.

See: https://github.com/makerdao/community/pull/2597/files#diff-46eac166654845e1c858fca919ba2d2ec7d5114fdc7248747924e2fc3b8b8e14R125-R127
and: https://github.com/makerdao/community/pull/2580/files#diff-a856ab68877ffd3cd18fa55b98aee86bfd37a7f0c1b7ce54bee1e43ee9634d00R102-R225

File: governance/votes/Executive Vote - April 4, 2024.md [hunk 11]
```diff
@@ -0,0 +1,92 @@
...
+
+### Approve TACO Resolutions
+
+- **Authorization:** $Missing?
```
Comment (0xldr) [inline governance/votes/Executive Vote - April 4, 2024.md:77]: :red_circle: **Blocking**

We understand this is pending and the Responsible Facilitator has been contacted.

**Review by 0xldr — CHANGES_REQUESTED** [review]
Please see our comments

File: governance/votes/Executive Vote - April 4, 2024.md [hunk 12]
```diff
@@ -0,0 +1,117 @@
...
+Updates Market Cap Parameters to the following:
+
+USDe 77% LLTV pool cap: **1 billion DAI**
+USDe 86% LLTV pool cap: **500 million DAI**
+USDe 91.5% LLTV pool cap: **200 million DAI**
+USDe 94.5% LLTV pool cap: **10 million DAI**
+sUSDe 77% LLTV pool cap: **1 billion DAI**
+sUSDe 86% LLTV pool cap: **200 million DAI**
+sUSDe 91.5% LLTV pool cap: **50 million DAI**
+sUSDe 94.5% LLTV pool cap: **10 million DAI**
```
Comment (0xldr) [inline governance/votes/Executive Vote - April 4, 2024.md:98]: :yellow_circle:  **medium**

This is all going on to one line when rendered in rich diff. The easiest fix is to make it bullet points.

```suggestion
* USDe 77% LLTV pool cap: **1 billion DAI**
* USDe 86% LLTV pool cap: **500 million DAI**
* USDe 91.5% LLTV pool cap: **200 million DAI**
* USDe 94.5% LLTV pool cap: **10 million DAI**
* sUSDe 77% LLTV pool cap: **1 billion DAI**
* sUSDe 86% LLTV pool cap: **200 million DAI**
* sUSDe 91.5% LLTV pool cap: **50 million DAI**
* sUSDe 94.5% LLTV pool cap: **10 million DAI**
```

File: governance/votes/Executive Vote - April 4, 2024.md [hunk 13]
```diff
@@ -0,0 +1,117 @@
...
+
+### Approve TACO Resolutions
+
+- **Authorization:** $Missing?
```
Comment (0xldr) [inline governance/votes/Executive Vote - April 4, 2024.md:102]:
```suggestion
- **Authorization:** [Forum post](https://forum.makerdao.com/t/bt-project-ethena-risk-legal-assessment/23978/9)
```

**Review by 0xldr — CHANGES_REQUESTED** [review]

File: governance/votes/Executive Vote - April 4, 2024.md [hunk 14]
```diff
@@ -0,0 +1,117 @@
+---
+title: Template - [Executive Vote] Increase GSM Pause Delay, Increase Spark MetaMorpho Vault Maximum Debt Ceiling, Add Native Vaults to Debt Ceiling Breaker, SparkLend Proxy Spell, and TACO Resolutions - April 4, 2024
```
Comment (VoteWizard) [inline governance/votes/Executive Vote - April 4, 2024.md:2]: We usually leave the spark proxy spell last, out of curiosity, why not this time?

File: governance/votes/Executive Vote - April 4, 2024.md [hunk 15]
```diff
@@ -0,0 +1,117 @@
...
+summary: Increase the Governance Security Module Pause Delay, increase Spark MetaMorpho Maximum Debt Ceiling, add native vaults to Debt Ceiling Breaker, Trigger SparkLend Proxy Spell, and approve BT Project Ethena Resolutions. 
+date: 2024-04-04T00:00:00.000Z
+address: "$spell_address"
+
+---
+# [Executive Proposal] Increase GSM Pause Delay, Increase Spark MetaMorpho Vault Maximum Debt Ceiling, Add Native Vaults to Debt Ceiling Breaker, SparkLend Proxy Spell, and TACO Resolutions
```
Comment (VoteWizard) [inline governance/votes/Executive Vote - April 4, 2024.md:8]:
```suggestion
# [Executive Proposal] Increase GSM Pause Delay, Increase Spark MetaMorpho Vault Maximum Debt Ceiling, Add Native Vaults to Debt Ceiling Breaker, SparkLend Proxy Spell, and TACO Resolutions - April 4, 2024
```

File: governance/votes/Executive Vote - April 4, 2024.md [hunk 16]
```diff
@@ -0,0 +1,117 @@
...
+* ETH-C
+* WSTETH-A
+* WSTETH-B
+* WBTC-A
+* WBTC-B
+* WBTC-C
+
+### Trigger SparkLend Proxy Spell
+
+- **Authorization:** Polls: [1](https://vote.makerdao.com/polling/QmdjqTvL), [2](https://vote.makerdao.com/polling/QmaEqEav), [3](https://vote.makerdao.com/polling/QmbCWUAP). [Forum Post](https://forum.makerdao.com/t/morpho-spark-dai-vault-update-1-april-2024/24006)
```
Comment (VoteWizard) [inline governance/votes/Executive Vote - April 4, 2024.md:70]:
```suggestion
- **Authorization:** Polls: [1](https://vote.makerdao.com/polling/QmdjqTvL), [2](https://vote.makerdao.com/polling/QmaEqEav), [3](https://vote.makerdao.com/polling/QmbCWUAP). [Forum Post](https://forum.makerdao.com/t/morpho-spark-dai-vault-update-1-april-2024/24006/9)  
```

**Comment by VoteWizard** [comment]
Great job, @MakerJanSky, and @0xldr! I've only found a couple of minor errors

File: governance/votes/Executive Vote - April 4, 2024.md (hunk 14 above)
Comment (0xldr) [reply governance/votes/Executive Vote - April 4, 2024.md:2]: In general we agree it should be last but this ordering matches the sheet and it was too late to change it at the point we were reviewing. In future Proxy Spell actions should go at the end of the copy - can you add this to the style guide?

File: governance/votes/Executive Vote - April 4, 2024.md [hunk 17]
```diff
@@ -0,0 +1,117 @@
...
+
+If you are new to voting in the Maker Protocol, please see the [voting guide](https://manual.makerdao.com/governance/voting-in-makerdao/on-chain-governance) to learn how voting works.
+
+---
+
+## Executive Summary
+
+If this executive proposal passes, the following actions will occur within the Maker Protocol:
+
+- The [Governance Pause Delay (GSM)](https://manual.makerdao.com/parameter-index/core/param-gsm-pause-delay) will be increased to **30 hours**.
```
Comment (0xldr) [inline governance/votes/Executive Vote - April 4, 2024.md:20]:
```suggestion
- The [Governance Security Module (GSM) Pause Delay](https://manual.makerdao.com/parameter-index/core/param-gsm-pause-delay) will be increased to **30 hours**.
```

File: governance/votes/Executive Vote - April 4, 2024.md [hunk 18]
```diff
@@ -0,0 +1,117 @@
...
+---
+
+## Executive Summary
+
+If this executive proposal passes, the following actions will occur within the Maker Protocol:
+
+- The [Governance Security Module (GSM) Pause Delay](https://manual.makerdao.com/parameter-index/core/param-gsm-pause-delay) will be increased to **30 hours**.
+- Spark MetaMorpho Vault DDM [Maximum Debt Ceiling (line)](https://manual.makerdao.com/module-index/module-dciam#maximum-debt-ceiling-line) will be increased to **1 billion DAI**.
+- Native Vault Engine ilks will be added to the [Debt Ceiling Breaker (LINE_MOM)](https://manual.makerdao.com/governance-processes/executive-vote-considerations/gsm-exceptions#debt-ceiling-breaker) as detailed below.
+- A SparkLend Proxy Spell [0x7748C5E6EEda836247F2AfCd5a7c0dA3c5de9Da2](https://etherscan.io/address/0x7748c5e6eeda836247f2afcd5a7c0da3c5de9da2) will be triggered.
```
Comment (0xldr) [inline governance/votes/Executive Vote - April 4, 2024.md:23]:
```suggestion
- The [Governance Security Module (GSM) Pause Delay](https://mips.makerdao.com/mips/details/MIP113#10-1-gsm-governance-security-module-pause-delay) will be increased to **30 hours**.
- Spark MetaMorpho Vault DDM [Maximum Debt Ceiling (line)](https://mips.makerdao.com/mips/details/MIP104#14-3-1-4-1-maximum-debt-ceiling-line-) will be increased to **1 billion DAI**.
- Native Vault Engine ilks will be added to the [Debt Ceiling Breaker (LINE_MOM)](https://mips.makerdao.com/mips/details/MIP113#10-2-3-debt-ceiling-breaker) as detailed below.
- A SparkLend Proxy Spell [0x7748C5E6EEda836247F2AfCd5a7c0dA3c5de9Da2](https://etherscan.io/address/0x7748c5e6eeda836247f2afcd5a7c0da3c5de9da2) will be triggered.
```

**Review by 0xldr — APPROVED** [review]

**Pull request merged**